build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/release/ReleaseToolsPlugin.java

```diff
@@ -47,6 +47,13 @@ public void apply(Project project) {
 
         final Version version = VersionProperties.getElasticsearchVersion();
 
+        project.getTasks()
+            .register(
+                "updateVersions",
+                UpdateVersionsTask.class,
+                t -> t.finalizedBy(project.findProject("server").getTasks().named("spotlessApply").get())
```

Comment:
Not sure if there's a more concise way to find the server spotless task here

Reply:
I don't think we want to use finalizedBy here. This isn't an appropriate use of `finalizedBy` (I wish these were never added to Gradle).

I'm also not sure why we would need to run this. Does this task not produce correctly formatted output? If not, then the tooling should just call this explicitly after doing the update.

Reply:
If we want to ensure that if running `./gradlew updateVersions spotlessApply` means that spotless runs after the version update then we can add an ordering rule `mustRunAfter("spotlessApply")`.

Reply:
I've changed it to `mustRunAfter`